eng/Packages.Data.props

```diff
@@ -38,14 +39,14 @@
     <PackageReference Update="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="2.6.1" />
     <PackageReference Update="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.2" />
     <PackageReference Update="Microsoft.CodeAnalysis" Version="2.3.0" />
-    <PackageReference Update="Microsoft.Extensions.Configuration.Abstractions" Version="2.2.0" />
-    <PackageReference Update="Microsoft.Extensions.Configuration.Binder" Version="2.2.0" />
+    <PackageReference Update="Microsoft.Extensions.Configuration.Abstractions" Version="2.1.0" />
```

Comment:
I've downgraded versions of dependencies to the current LTS version. Azure.Core.Extensions is the only project using them